src/rgw/driver/rados/rgw_tools.cc

```diff
@@ -54,13 +54,12 @@ int rgw_init_ioctx(const DoutPrefixProvider *dpp,
 
     if (mostly_omap) {
       // set pg_autoscale_bias
-      bufferlist inbl;
       float bias = g_conf().get_val<double>("rgw_rados_pool_autoscale_bias");
       int r = rados->mon_command(
 	"{\"prefix\": \"osd pool set\", \"pool\": \"" +
 	pool.name + "\", \"var\": \"pg_autoscale_bias\", \"val\": \"" +
 	stringify(bias) + "\"}",
-	inbl, NULL, NULL);
+	{}, NULL, NULL);
```

Comment:
If already touching this specific line: nullptr (here and below)

Reply:
No, sorry. Changing NULL to nullptr is not what this PR is about, and I don't agree with mixing this into one unrelated commit, because I consider this confusing. And I don't want to spend time now to amend this PR with a random unrelated change.

Reply:
OK - your call.